### PR TITLE
[modules] pin cihub/seelog version

### PR DESCRIPTION
**Pull request by remeh**
### What does this PR do?

The `master` version of seelog is misplacing the line numbers in logs.

### Motivation

Logs are barely usable for debug with misplaced lines number.

### Describe your test plan

Run the Datadog Agent, check that some `file.go:line-number` are matching.

### CODE REVIEW

**Review by olivielpeau — APPROVED** [review]
LGTM!

Also confirmed on my end that pinning to this seelog commit/version (`d2c6e5aa9fbf`/`2.6`) fixes the issue

File: go.mod [hunk 1]
```diff
@@ -18,6 +18,7 @@ replace (
 
 // Internal deps fix version
 replace (
+	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf
```
Comment (clamoriniere) [inline go.mod:21]: does setting `v2.6` didn't works?

Comment (remeh) [reply go.mod:21]: Unfortunately not, most likely because it's not following the `vx.y.z`

Comment (clamoriniere) [reply go.mod:21]:
```suggestion
	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf // v2.6
```

**Review by clamoriniere — APPROVED** [review]

**Review by albertvaka — APPROVED** [review]
Was this bumped during the go.mod migration?

**Pull request merged**